tests/artist-delete.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('delete artist', () => {
+  let db;
+  let artists;
+  beforeEach(async () => {
+    db = await getDb();
+    await Promise.all([
+      db.query('INSERT INTO Artist (name, genre) VALUES(?, ?)', [
+        'Periphery',
+        'djent',
+      ]),
```

Comment:
```suggestion
      db.query(`INSERT INTO Artist (name, genre) VALUES('Periphery', 'djent')`),
```
See main PR comment for details